kroxylicious-kms-provider-aws-kms/src/test/java/io/kroxylicious/kms/provider/aws/kms/AwsKmsEdekSerdeTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.kms.provider.aws.kms;
+
+import java.nio.ByteBuffer;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class AwsKmsEdekSerdeTest {
+
+    private final AwsKmsEdekSerde serde = new AwsKmsEdekSerde();
+
+    static Stream<Arguments> keyRefs() {
+        return Stream.of(
+                Arguments.of("keyref", "1234abcd-12ab-34cd-56ef-1234567890ab"));
```

Comment:
Do you have plans for more elements in the stream? Feels a bit odd have a single element stream why not just have it in the test case?

Reply:
No, it was a hang-over from the Vault classes I copied from.